### ISSUE TITLE
zIndex is not working on IOS

### ISSUE THREAD

**Issue by sleeviin** [react-community/react-native-maps#2133]
### Is this a bug report?

YES

### Have you read the [Installation Instructions](https://github.com/react-community/react-native-maps/blob/master/docs/installation.md)?

YES

### Environment
react-native: 0.51.0
react: 16.0.0
react-native-maps: 0.20.1
I'm using iPhone 6s with the latest IOS Version

### Steps to Reproduce

I'm using Maps in order to show a lot of dynamic items, but some of these items needs to be always on Top, and now it doesn't work correctly, sometimes it works but mostly not, zIndex looks like is not working fine.

On android it works fine using elevation instead of zIndex, but I need to make this happen on IOS.

In fact, I'm using the example file provided by React Native Maps, and also it doesn't works.

I need a solution.

### Expected Behavior

all special items should be on top, and other markers shouldn't cover these items.

### Actual Behavior

zIndex looks like doesn't work, all markers are with the same rules, special items are like equals with normal items.

### Reproducible Demo

https://snack.expo.io/ByX2jYOqf

**Comment by martsie** [comment]
There are some other issues that mention this:

https://github.com/react-community/react-native-maps/issues/1788
https://github.com/react-community/react-native-maps/issues/1671

**I think I have a hacky solution**
1. First I order markers in the zIndexes I want them to be in (so make sure you sort your markers array before mapping it to the screen).
2. Whenever you update markers, loop through your markers and set zIndexes, eg in my code in componentDidUpdate
```
// mapMarkerViews were ref'd when adding markers to this.mapMarkerViews.
markers.forEach((marker, index) => {
      const mapMarkerView = this.mapMarkerViews[marker.id];
      if (mapMarkerView) {
        mapMarkerView.setNativeProps({ zIndex: index });
      }
    });
```
3. Animate to the current map coordinates with a very minor change in order to trigger the map to update:
```
this.map.animateToCoordinate({ latitude: currentLat + 0.0001, longitude: currentLongitude });
```

Basically what we're doing is 1. Setting the initial order of the markers using array order. 2 Updating native zIndex props. 3. Triggering a map update.

Hopefully this helps someone.

**Comment by fafnir0007** [comment]
const style = StyleSheet.create({
  dinamicItems:{
   zIndex:1,
   position:'absolute'
  }
})

export default style

**Comment by urikphytech** [comment]
I had this issue with overlapping polygons and using @martsie solution (just step number 2) did the trick for me. Thanks

**Comment by alvelig** [comment]
Duplicate of #1671
